### ISSUE TITLE
Theme variants using FontAwesome

### ISSUE THREAD

**Issue by xpoke** [mumble-voip/mumble-theme#17]
Hey - It's been a while since I created the theme, but I saw you guys released 1.3 (!)

I was wondering if there's an interest in making variants using the icons from FontAwesome's GPL-friendly library.
https://fontawesome.com/
https://github.com/FortAwesome/Font-Awesome
https://github.com/FortAwesome/Font-Awesome/blob/master/LICENSE.txt

I specifically think some users would like a less colorful theme, and we can easily add high quality icons.
Let me know if this fits with your plans for a 1.3.1 release

**Comment by streaps** [comment]
These new added icons are inconsistent with the 1.3 theme:

https://github.com/mumble-voip/mumble-theme/blob/master/ear.svg
https://github.com/mumble-voip/mumble-theme/blob/master/lock_locked.svg
https://github.com/mumble-voip/mumble-theme/blob/master/lock_unlocked.svg

It would be nice to have a consistent theme for 1.4 too. Maybe that could be based on FontAwesome. I like the colorful style though ;)

**Comment by xpoke** [comment]
Hey - I no longer have the same set up as I used to when creating these (Adobe Illustrator specifically). I'll see what I can do but if you have ideas please let me know

**Comment by toby63** [comment]
@xpoke:
Regarding the locks, 
* locked state: imo it should be filled with colour, also no keyhole is necessary.
* unlocked state: It should look more like this: https://s3.envato.com/files/235925363/Access%20Granted%20590x332.jpg
:thinking: But overall maybe the open lock is not visible enough, though the green colour might be enough...
An alternative would be a key symbol or something similar.

The ear is more difficult.
@Krzmbrzl could you explain the purpose of the ear?
Maybe we could find a better icon for this.

Regarding the overall theme:
I like it very much :+1:.
The only thing I would change is the talking indicator (this (and the related ones): https://raw.githubusercontent.com/mumble-voip/mumble-theme/master/talking_on.svg )
I don't like this figure.
In my [concept for a "new" UI](https://github.com/mumble-voip/mumble/issues/3608#issuecomment-601931764) (based on your icons) I used a "classic" microphone instead:
https://user-images.githubusercontent.com/61703250/77208584-678fc680-6afc-11ea-9aa8-8d53c99c0763.png

But maybe you have an even better idea.

**Comment by Krzmbrzl** [comment]
> @Krzmbrzl could you explain the purpose of the ear?

It marks listener proxies of the ChannelListener feature. Aka. this is a virtual ear that a user can drop into a channel in order to hear what is being spoken in it without having to join it.

**Comment by Kissaki** [comment]
I feel like Mumble/the UI is getting way too complex. Who’s supposed to understand all that? 🤔

**Comment by toby63** [comment]
> I feel like Mumble/the UI is getting way too complex. Who’s supposed to understand all that? thinking

You are right, maybe you should switch back to "extended options" or so in the future.

Regarding this issue:
Especially this lock thing seems weird to me.

**Comment by Krzmbrzl** [comment]
@toby63 the locks where introduced in https://github.com/mumble-voip/mumble/pull/3929

**Comment by streaps** [comment]
> I feel like Mumble/the UI is getting way too complex. Who’s supposed to understand all that?

It was already too complex before all the new features were added. But not only do we have feature creep, the UI is also getting uglier. These new icons are inconsistent with the theme. Now we have new features, but nobody seems to be responsible for the UI/design. Is there any plan for this or are you really seriously want to release this crap with 1.4?

I know that icons don't generate themselves and the project doesn't have a designer / UX guy. I also would do some ugly icons first, if I were writing code involving the UI. Unfortunately this is an application that has a GUI which is used by users. You care a lot about the code before it gets merged, but UI uglification should equally be a blocker for a merge.

But I guess this is super "OT" in mumble-theme (and this issue) and should be better discussed in the mumble repo. ;-P

Back to topic: if a new theme were introduced, will it be maintained in the future or will it be inconsistent again with the following release?

**Comment by Krzmbrzl** [comment]
> Is there any plan for this or are you really seriously want to release this crap with 1.4?

> You care a lot about the code before it gets merged, but UI uglification should equally be a blocker for a merge.

How about you stop complaining and simply take on the work of actually improving stuff? Obviously you seem to have an idea on how these icons should look. Then go ahead and create them.
If that doesn't happen, then yes it's going to be released that way. That's how OpenSource works: Everyone contributes to the parts that (s)he can do best and if there is nobody that wants to / can do fancy icons, then we have to settle with what we get.

> But I guess this is super "OT" in mumble-theme (and this issue) and should be better discussed in the mumble repo. ;-P

The icons are part of the theme so this repo is the exact place for this kind of discussion. It is however OT for this particular issue :shrug:

**Comment by toby63** [comment]
@streaps I would also like to hear a bit more substantial argumentation than just complaining about how everything is bad and not explaining what exactly is bad.
I agree that it's not so intuitive, but I think in part that is because of "missing" or overcomplex functionality (for example, 1. the ACL system could be easier  2. as you suggested the audio settings could maybe be applied more automatically) and in other parts it is because things are simply a bit more complicated, so maybe you could clarify 1. what is bad in your oppinion 2. and how it can be solved.

Nonetheless I agree that you (@Krzmbrzl and the rest of the team) might work a bit closer with @xpoke and maybe others who would like to participate in UI and themes, instead of just applying concept icons without any notice that these should maybe be "polished".
No offence, this is a just a friendly suggestion from my side :slightly_smiling_face: .
An example for a procedure: You apply a new functionality (or something different) that requires UI changes -> you apply a concept -> then open an issue (maybe here) to discuss the change or in case someone wants to work on that concept or icon etc. to make it look better.

**Comment by Krzmbrzl** [comment]
All icons went in via a PR. As nobody complained about them, they got merged eventually. Besides: I was under the impression that xpoke didn't actively work on this anymore :shrug: 

> Maybe mumble-voip/mumble#3057 would be a better solution.

Why should channels be hidden just because you haven't configured your access tokens accordingly yet? I think this is a bad idea. But this is also 100% OT here. If you want to discuss this further, please do so in the PR for this feature :)

> Or you could just show a lock for locked channels, and all other channels have no lock-icon.

That's the case. The open lock is used for channels that have access restrictions (e.g. a password) set but which you can still enter due to your access tokens / group. If you removed the indicator completely, you might be confused as to why other users can't join you in that channel.

**Comment by toby63** [comment]
> All icons went in via a PR. As nobody complained about them, they got merged eventually.

But you must have seen for yourself that the design is not so "very" good (no offence; I am also no graphic designer).
So in such a case, it would be good to open an issue and ask whether someone wants to improve the design.

> But this is also 100% OT here.

It's not, if a different solution is applied, there is no need for the lock icons, so it is 100% related.

> That's the case. The open lock is used for channels that have access restrictions (e.g. a password) set but which you can still enter due to your access tokens / group. If you removed the indicator completely, you might be confused as to why other users can't join you in that channel.

That is completely confusing.
Also isn't this going to be changed? I read somewhere that you want to switch to access tokens only.

**Comment by Krzmbrzl** [comment]
> But you must have seen for yourself that the design is not so "very" good (no offence; I am also no graphic designer).

I don't. I think these icons are alright.

> So in such a case, it would be good to open an issue and ask whether someone wants to improve the design.

They can be changed as soon as someone is bothered by it and creates better ones. No need to let that delay the merge of a new feature.

> It's not, if a different solution is applied, there is no need for the lock icons, so it is 100% related.

This issue is about using FontAwesome - 100% unrelated

> That is completely confusing.
Also isn't this going to be changed? I read somewhere that you want to switch to access tokens only.

I'll only answer to discussion about this in https://github.com/mumble-voip/mumble/pull/3929

**Comment by toby63** [comment]
> I think these icons are alright.

The green is not even the same green used elsewhere.
Anyway imo this should work differently in the future.

> They can be changed as soon as someone is bothered by it and creates better ones. No need to let that delay the merge of a new feature.

Thats exactly where I (and maybe also streaps) disagree. 
A consistent UI and theme is very important and is not a thing to bother with, when someone opens an issue.
It should be taken care of continuously.

> This issue is about using FontAwesome - 100% unrelated

nit-picking, xpoke already responded to that, but nonetheless I agree maybe we should seperate some topics from here.
